docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -42,7 +48,27 @@ To create a notification using property, follow these steps:
     notification.Post(3000);
     ```
 
-3. Respond to OnWindowTouch:
+3. Set the notification content:
+
+    ```cs
+    // Sets text in Notification
+    TextLabel text = new TextLabel("This is notification sample.");
+    text.HorizontalAlignment = HorizontalAlignment.Center;
+    text.VerticalAlignment = VerticalAlignment.Center;
+    text.TextColor = Color.Black;
+    text.Size2D = new Size2D(720, 100);
+
+    // Add text label to view.
+    view.Add(text);
+
+    // Add view to the main Window.
+    Window.Instance.Add(view);
+
+    // Post notification for 3000ms.
+    notification.Post(3000);
+    ```
+
+4. Respond to `OnWindowTouch`:

```

Comment:
@patrykka

Is `OnWindowTouch` an event?  or what? 


 4. Respond to `OnWindowTouch`...........???:

Reply:
Yes, this is event.
Respond to OnWindowTouch event

Reply:
@patrykka To this dialogue I have additional questions.

It seems the `OnWindowTouch` is defined event, but in this page defining is not written.

Also when I checked in the API reference, https://samsung.github.io/TizenFX/API8/api/Tizen.NUI.Components.Notification.html,
I couldn't find SetDismissOnTouch method.

Would you please check?

Reply:
Once you respond to @mijin85cho query, then please incorporate the following (if it is still an event).
```suggestion 
4. Respond to `OnWindowTouch` event: 
```

Reply:
@mijin85cho  @nawab-samsung 
 OnWindowTouch is Window event - screen ( I left that from previous description, while in example  -> https://github.com/Samsung/Tizen-CSharp-Samples/pull/194 I used some kind of "timer" to show notification on each 5 seconds.

Regarding SetDismissOnTouch  - looks this is public function with marked "[EditorBrowsable(EditorBrowsableState.Never)]"
This mean that it is not published in Components API reference. What I should do with it ?

Reply:
OnWindowTouch is Window event - screen : 
It seems "OnWindowTouch" is not a specific function or method with name "OnWindowTouch", but just mentioning an event notion.
If so, please remove the code tag. This line seems to mean "Respond to the touch event on window", not "Respond to an event with a name, OnWindowTouch "


Regarding SetDismissOnTouch : 
Notification guide will be published on docs.tizen.org on public. 
So if there is a non published method to the public, please don't use it.
I'm concerned about the case that developers in outside try to follow the guide but can't find that method.

If I understood something wrong, please let me know.

Reply:
ok, this point was related only with dismissing notification, point 4 deleted